ts/components/messages/MessageListComponent.tsx

```diff
@@ -58,6 +59,8 @@ class MessageListComponent extends React.Component<Props> {
     );
   };
 
+  flatListRef: FlatList<MessageState> | null | undefined;
```

Comment:
Try creating ref using React.createRef.

https://reactjs.org/docs/refs-and-the-dom.html#creating-refs